mathics_scanner/data/named-characters.yml

```diff
@@ -5003,13 +4992,15 @@ Information:
   is-letter-like: false
   operator-name: Information
 
+# While one might think that Integral is an WL operator, it is not.
```

Comment:
> ### Precedence of Integration Operators
>
> Forms such as involve two operators: [\[Integral]](https://reference.wolfram.com/language/ref/ character/Integral.html) 
> and [\[DifferentialD]](https://reference.wolfram.com/language/ref/character/DifferentialD.html). 
> The former has a precedence just above , while the latter has one > just below [Power](https://reference.wolfram.com/language/ref/Power.html). 
> The precedence of determines when needs to be parenthesized, while the precedence of determines when needs to be parenthesized. Note that the form does not involve , and therefore behaves like a normal prefix operator in this case.
> [∮](https://reference.wolfram.com/language/ref/character/ContourIntegral.html), [∲](https://> reference.wolfram.com/language/ref/character/ClockwiseContourIntegral.html) and 
> [∯](https://reference.wolfram.com/language/ref/character/DoubleContourIntegral.html) work the same as [∫](https://reference.wolfram.com/language/ref/character/Integral.html).
> See ["Two-Dimensional Input Forms"](https://reference.wolfram.com/language/tutorial/InputSyntax.html#15273) for two‐dimensional input forms associated with integration operators.

(from https://reference.wolfram.com/language/tutorial/OperatorInputForms.html)

Reply:
Ok. Thanks for the information and corrections.

I will revise this and the spelling corrections below soon. The thing about Integral then is that its "arity" is special- neither Prefix, Postfix, Infix, or MatchFix. I'll create a new category "Integral" then.